src/sota_tools/CMakeLists.txt

```diff
@@ -98,17 +98,16 @@ add_dependencies(build_tests garage-deploy)
 
 
 install(TARGETS garage-deploy RUNTIME DESTINATION bin COMPONENT garage_deploy)
-add_custom_target(download_garage_sign COMMAND wget --no-check-certificate https://ats-tuf-cli-releases.s3-eu-central-1.amazonaws.com/cli-0.2.0-29-gf6f095a.tgz -O ${CMAKE_CURRENT_BINARY_DIR}/garage-sign.tgz
-COMMAND tar -xzf ${CMAKE_CURRENT_BINARY_DIR}/garage-sign.tgz -C ${CMAKE_CURRENT_BINARY_DIR}
-COMMAND chmod +x ${CMAKE_CURRENT_BINARY_DIR}/garage-sign/bin/*)
+execute_process(COMMAND wget --no-check-certificate https://ats-tuf-cli-releases.s3-eu-central-1.amazonaws.com/cli-0.2.0-29-gf6f095a.tgz -O ${CMAKE_CURRENT_BINARY_DIR}/garage-sign.tgz)
```

Comment:
Can we upgrade this to `https://ats-tuf-cli-releases.s3-eu-central-1.amazonaws.com/cli-0.2.0-52-g83fbfe6.tgz`? There have been a number of important fixes that we should be using.